src/js/plugin.js

```diff
@@ -312,6 +312,9 @@ class Plugin {
     if (name !== BASE_PLUGIN_NAME) {
       if (Plugin.isBasic(plugin)) {
         Player.prototype[name] = createBasicPlugin(name, plugin);
+        Object.keys(plugin).forEach(function(prop) {
+          Player.prototype[name][prop] = plugin[prop];
+        });
```

Comment:
I thoughts that too at first, but `Player.prototype[name]` does not exist until we get the return value from `createBasicPlugin`